pkgs/build-support/rust/default.nix

```diff
@@ -48,14 +48,7 @@ in stdenv.mkDerivation (args // {
     chmod -R +w "$cargoDepsCopy"
 
     mkdir .cargo
-    cat >.cargo/config <<-EOF
-      [source.crates-io]
-      registry = 'https://github.com/rust-lang/crates.io-index'
-      replace-with = 'vendored-sources'
-
-      [source.vendored-sources]
-      directory = '$(pwd)/$cargoDepsCopy'
-    EOF
```

Comment:
What does cargo-vendor now generate instead?

Reply:
The current stdout output for cargo-vendor 0.13 on alacritty is:
```
To use vendored sources, add this to your .cargo/config for this project:

    [source.crates-io]
    replace-with = "vendored-sources"
    
    [source."https://github.com/jwilm/libfontconfig"]
    git = "https://github.com/jwilm/libfontconfig"
    branch = "updated-2017-10-8"
    replace-with = "vendored-sources"
    
    [source."https://github.com/jwilm/rust-fontconfig"]
    git = "https://github.com/jwilm/rust-fontconfig"
    branch = "updated-2017-10-8"
    replace-with = "vendored-sources"
    
    [source.vendored-sources]
    directory = "/home/zimbatm/go/src/github.com/jwilm/alacritty/vendor"
```

I remember submitting a PR to send the comment on stderr and remove the indent as well, not sure why it was reverted.

Reply:
> I remember submitting a PR to send the comment on stderr and remove the indent as well, not sure why it was reverted.

It wasn't; this PR relies on that. Perhaps the indentation is on stderr too?